dist/index.html

```diff
@@ -5,7 +5,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>To Do List</title>
-  <link rel="icon" href="../lista-de-deseos.png"> <!-- Web Icon -->
+  <link rel="icon" href="C:\Users\manue\Documents\To-do-list\src\assets\lista-de-deseos.png"> <!-- Web Icon -->
   <script src="https://kit.fontawesome.com/5211805414.js" crossorigin="anonymous"></script> <!-- Font Awesome -->
 <script defer src="runtime.js"></script><script defer src="main.js"></script></head>
 <body>
```

Comment:
- I can see that you have implemented some of the project requirements but the user interface isn't showing any of them. The add task, delete task, edit task, and clear all completed button isn't working. Kindly check if you are linking your javascript files correctly to your HTML and please implement the changes.